src/components/events/details/EventAccessDetails.vue

```diff
@@ -58,7 +58,7 @@ export default {
       tableHeadLinks: [
         {
           name: 'Edit',
-          routeName: 'EventCreate',
+          routeName: 'E',
```

Comment:
Why you changed it by 'E', could you use a meaningful word for this?

Reply:
My mistake, fixed and pushed.